<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QDialog" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROFILE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 22, 40);
</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#E5CB94;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 203, 148);
</string>
   </property>
   <property name="text">
    <string>RECENT LOGINS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2
{
	border:none;
	border-radius:16 px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3 px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>26</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(229, 203, 148);
</string>
        </property>
        <property name="text">
         <string>CHANGE USERNAME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEdit_username
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#lineEdit_username:hover
{
	border:none;
	background:#E5CB94;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(229, 203, 148);
</string>
        </property>
        <property name="text">
         <string>CHANGE PASSWORD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEdit_password
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#lineEdit_password:hover
{
	border:none;
	background:#E5CB94;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(229, 203, 148);
</string>
        </property>
        <property name="text">
         <string>CURRENT PASSWORD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_current_pass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineEdit_current_pass
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#lineEdit_current_pass:hover
{
	border:none;
	background:#E5CB94;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>221</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_EDIT">
        <property name="styleSheet">
         <string notr="true">#pushButton_EDIT
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 5px;
}

#pushButton_EDIT:hover
{
	border:none;
	background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}
</string>
        </property>
        <property name="text">
         <string>EDIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="FRAME_DELETE">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <bold>true</bold>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">

	border:none;
	color:#E5CB94;
	


</string>
         </property>
         <property name="text">
          <string>DELETE ACCOUNT</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>97</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DELETE">
         <property name="styleSheet">
          <string notr="true">#pushButton_DELETE{
	image: url(:/resources/resources/delete-icon-980x980.png);
border:none;
	background: rgb(170, 0, 0);;
	color:#E5CB94;
	border-radius:16px;
	border: 2px solid red ;
border-color: #E5CB94;
  border-radius: 6px;
  padding: 3 px;
}
#pushButton_DELETE:hover
{
	border:none;
	background:#E5CB94;
	color:#0F1628;
	border-radius:16 px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 6px;
  padding: 3 px;
}


</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_WELCOME">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>67</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 203, 148);
</string>
   </property>
   <property name="text">
    <string>PROFILE INFORMATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BACK">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>460</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_BACK
{
	border:none;
	background:#0F1628;
	color:#E5CB94;
	border-radius:16px;
	border: 1px solid yellow ;
border-color: #E5CB94;
  border-radius: 12px;
  padding: 3px;
}

#pushButton_BACK:hover
{
	border:none;
background:floral white;
	color:#0F1628;
	border-radius:16px;
	border: 2px solid blue ;
border-color: #0F1628;
  border-radius: 12px;
  padding: 1px;
}

</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_strength">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>202</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_picture">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>54</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_picture
{
	image: url(:/resources/resources/kindpng_1712282.png);
	border:none;
	background:#E5CB94;
	color:#0F1628;;
	border-radius:16px;
	border: 1px solid blue ;
border-color: #0F1628;
  border-radius: 20px;
  padding: 3px;
}


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
